x11-misc/polybar/polybar-3.3.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
You should use repoman to commit so it fixes this old header for you automatically :)

Reply:
Done, thanks for suggestion!